src/pdn/test/power_switch.ok

```diff
@@ -40,8 +40,8 @@ Straps:
     Offset: 13.6000
     Snap to grid: false
 Connect:
-  Connect layers met1 -> met4
-  Connect layers met4 -> met5
+  Connect layers met1 -> met4 (1)
+  Connect layers met4 -> met5 (2)
```

Comment:
what is the purpose of the numbers in ()?

Reply:
in what scenario does the ordering matter?

Reply:
I'll remove it, I don't think ordering matters